src/lib/fmt/ThreadIdFormatter.hxx

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: BSD-2-Clause
+// author: Max Kellermann <max.kellermann@gmail.com>
+
+#ifndef THREAD_ID_FORMATTER_HXX
+#define THREAD_ID_FORMATTER_HXX
+
+#include <fmt/format.h>
+#include <sstream>
```

Comment:
Please please please no iostreams in MPD!
Why do we even need this whole header??